### ISSUE TITLE
PopoverField should be a menu, not a tooltip

### ISSUE THREAD

**Issue by NightJar** [silverstripe/silverstripe-admin#352]
in regards to ARIA and the `role` of the associated Overlay component.

This stems from the use of a component designed as a tool-tip as a menu.
https://github.com/silverstripe/silverstripe-admin/blob/1.0/client/src/components/PopoverField/PopoverField.js#L41
To fix this may require a rework of the component as a whole, since it leverages existing external code:
https://github.com/open-sausages/react-bootstrap/blob/master/src/Popover.js#L96

This core component affects:
https://github.com/silverstripe/silverstripe-asset-admin/blob/1.0/client/src/components/BulkActions/BulkActions.js
and blocks:
https://github.com/symbiote/silverstripe-gridfieldextensions/pull/225#pullrequestreview-75982197
probably among other components that use core as an example.

![image](https://user-images.githubusercontent.com/778003/32707421-92a1af1e-c888-11e7-97ac-fee6575f0081.png)

**Comment by chillu** [comment]
Good point. We'll likely fix this as part of switching the UI to React in the first place. For now, both existing implementations ("more options" in file edit view, and "bulk actions" in file overview) are keyboard accessible - while your implementation isn't. So from my perspective, this doesn't block getting https://github.com/symbiote/silverstripe-gridfieldextensions/pull/225 accessible.
